packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -491,16 +491,27 @@ export const audiusBackend = ({
     )
   }
 
-  async function sanityChecks(audiusLibs: any) {
+  async function sanityChecks(
+    audiusLibs: any,
+    args: { wallet?: string; handle?: string }
+  ) {
     try {
       const sanityChecks = new SanityChecks(audiusLibs)
-      await sanityChecks.run()
+      await sanityChecks.run(args)
     } catch (e) {
       console.error(`Sanity checks failed: ${e}`)
     }
   }
 
-  async function setup() {
+  async function setup({
+    wallet,
+    userId,
+    handle
+  }: {
+    wallet?: string
```

Comment:
can you add a comment here explaining why all 3 are optional and what they impact?